test/e2e/util.go

```diff
@@ -1721,6 +1722,14 @@ func getNodeEvents(c *client.Client, nodeName string) []api.Event {
 	return events.Items
 }
 
+func ListSchedulableNodesOrDie(c *client.Client) *api.NodeList {
+	nodes, err := c.Nodes().List(unversioned.ListOptions{FieldSelector: unversioned.FieldSelector{fields.Set{
+		"spec.unschedulable": "false",
+	}.AsSelector()}})
+	expectNoError(err, "Failed to list nodes for e2e cluster.")
```

Comment:
Since you called it "OrDie" this should probably by some failure - not just expect NoError.